WordPress/Classes/Services/LoginFacade.m

```diff
@@ -62,6 +60,28 @@ - (void)requestOneTimeCodeWithLoginFields:(LoginFields *)loginFields
     }];
 }
 
+- (void)loginToWordPressDotComWithGoogleIDToken:(NSString *)googleIDToken
+{
+    if ([self.delegate respondsToSelector:@selector(displayLoginMessage:)]) {
+        [self.delegate displayLoginMessage:NSLocalizedString(@"Connecting to WordPress.com", nil)];
```

Comment:
I know we already do this in `-signInToWordpressDotCom` but I don't really love having UI-surfaced strings in the networking code. If instead we told the delegate the state had changed, the VC could decide how to present it: string, icon, or otherwise. Think that's worth changing?

Reply:
I'm not too much of a fan either.  I still have the dream of nuking the facade completely, but until then I think we should be consistent.